service/runtime/manager/events.go

```
@@ -117,7 +118,8 @@ func (m *manager) processEvent(key string) {
 		err = runtime.Update(ev.Service, gorun.UpdateNamespace(ns))
 	case gorun.Create:
 		// generate an auth account for the service to use
-		acc, err := m.generateAccount(ev.Service, ns)
```

Error was being created here as a local variable which caused create errors to be swallowed.